packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3236,6 +3014,233 @@ export default class Meeting extends StatelessWebexPlugin {
+            })) ||
+          // @ts-ignore
+          !this.config.experimental.enableUnifiedMeetings ||
+          this.isLocusCall(),
+        canDisableVideo: ControlsOptionsUtil.hasHints({
+          requiredHints: [DISPLAY_HINTS.DISABLE_VIDEO],
+          displayHints: this.userDisplayHints,
+        }),
+        canShareFile:
+          (ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.SHARE_FILE],
+            displayHints: this.userDisplayHints,
+          }) &&
+            (ControlsOptionsUtil.hasPolicies({
+              requiredPolicies: [SELF_POLICY.SUPPORT_FILE_SHARE],
+              policies: this.selfUserPolicies,
+            }) ||
+              // @ts-ignore
+              !this.config.experimental.enableUnifiedMeetings)) ||
+          this.isLocusCall(),
+        canTransferFile: ControlsOptionsUtil.hasPolicies({
+          requiredPolicies: [SELF_POLICY.SUPPORT_FILE_TRANSFER],
+          policies: this.selfUserPolicies,
+        }),
+        canShareApplication:
+          ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.SHARE_APPLICATION],
+            displayHints: this.userDisplayHints,
+          }) &&
+          ControlsOptionsUtil.hasPolicies({
+            requiredPolicies: [SELF_POLICY.SUPPORT_APP_SHARE],
+            policies: this.selfUserPolicies,
+          }),
+        canShareCamera:
+          ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.SHARE_CAMERA],
+            displayHints: this.userDisplayHints,
+          }) &&
+          ControlsOptionsUtil.hasPolicies({
+            requiredPolicies: [SELF_POLICY.SUPPORT_CAMERA_SHARE],
+            policies: this.selfUserPolicies,
+          }),
+        canShareDesktop:
+          (ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.SHARE_DESKTOP],
+            displayHints: this.userDisplayHints,
+          }) &&
+            (ControlsOptionsUtil.hasPolicies({
+              requiredPolicies: [SELF_POLICY.SUPPORT_DESKTOP_SHARE],
+              policies: this.selfUserPolicies,
+            }) ||
+              // @ts-ignore
+              !this.config.experimental.enableUnifiedMeetings)) ||
+          this.isLocusCall(),
+        canShareContent:
+          ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.SHARE_CONTENT],
+            displayHints: this.userDisplayHints,
+          }) || this.isLocusCall(),
+        canAnnotate: ControlsOptionsUtil.hasPolicies({
+          requiredPolicies: [SELF_POLICY.SUPPORT_ANNOTATION],
+          policies: this.selfUserPolicies,
+        }),
+        canUseVoip:
+          ControlsOptionsUtil.hasHints({
+            requiredHints: [DISPLAY_HINTS.VOIP_IS_ENABLED],
+            displayHints: this.userDisplayHints,
+          }) &&
+          ControlsOptionsUtil.hasPolicies({
+            requiredPolicies: [SELF_POLICY.SUPPORT_VOIP],
+            policies: this.selfUserPolicies,
+          }),
```

Comment:
Do I need any of the logic in e.g. canShareDesktop? I'm not sure what the logic represents or why it is only for some hints

Reply:
Logic is there because policy does not work for v1 meetings and one to one calls. canShareDesktop and other have added exceptions for such cases. So even after your change for checking policy v1/ one to one calls should still work.

Reply:
is there a reason it's not got exceptions for canShareApplication/canShareCamera?

Reply:
It's there for canShareApplication but not there for canShareCamera, may be canShareCamera feature isn't there for v1 - just guessing